Products/Brass-Beta/gWASM.md

```diff
@@ -39,19 +39,353 @@ If you would like to share your thoughts about **gWASM** please fill out this **
 
 ### Quick start - testing gWASM locally
 
+?> This tutorial is based on our Devcon5 gWASM workshops `hello world!` example. If you would like to set up your own Golem Unlimited cluster then follow instructions at [devcon5 worksops page](https://devcon.golem.network)
+
 In this quick tutorial you will
 
-* Install latest release of Golem
-* Run two nodes on the same machine (one being a Requestor, and second one being Provider) 
+* Install Rustlang (*stable version 1.38.0*) and emscripten (*1.28.45-fastcomp*)
+* Download latest gWASM runner (Command line tool for running gWasm compatible apps locally, via Golem Unlimited or via Brass Golem.)
 * Run simple `hello world` task
 
+
+---
+
+#### 1. Prerequisites
+
+Make sure that you have **Rustlang** and **emscripten** installed on your machine. 
+
+**2.1.** [Rust](https://www.rust-lang.org/tools/install) - **stable version 1.38** 
+
+open up your console/terminal and paste:
+
+**2.1.1. Installation:**
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+**2.1.2. Toolchain version:**
+
+```bash
+rustup toolchain add 1.38.0
```

Comment:
Hmm, why are we forcing version `1.38.0` here?

Reply:
By @prekucki recommendation, as this stable of toolchain was tested and works properly with emscripten fastcomp backend `1.38.45-fastcomp`